org.apache.zeppelin.helium.ClassLoaderApplication.run : $r2 = staticinvoke <java.lang.Thread: java.lang.Thread currentThread()>() >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : r3 = virtualinvoke $r2.<java.lang.Thread: java.lang.ClassLoader getContextClassLoader()>()
org.apache.zeppelin.helium.ClassLoaderApplication.run : r3 = virtualinvoke $r2.<java.lang.Thread: java.lang.ClassLoader getContextClassLoader()>() >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r6 = staticinvoke <java.lang.Thread: java.lang.Thread currentThread()>()
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r6 = staticinvoke <java.lang.Thread: java.lang.Thread currentThread()>() >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r7 = r0.<org.apache.zeppelin.helium.ClassLoaderApplication: java.lang.ClassLoader cl>
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r7 = r0.<org.apache.zeppelin.helium.ClassLoaderApplication: java.lang.ClassLoader cl> >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r6.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>($r7)
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r6.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>($r7) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r10 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r6.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>($r7) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r11 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r6.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>($r7) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r6.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>($r7) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r8 = r0.<org.apache.zeppelin.helium.ClassLoaderApplication: org.apache.zeppelin.helium.Application app>
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r8 = r0.<org.apache.zeppelin.helium.ClassLoaderApplication: org.apache.zeppelin.helium.Application app> >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r10 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r8 = r0.<org.apache.zeppelin.helium.ClassLoaderApplication: org.apache.zeppelin.helium.Application app> >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r11 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r8 = r0.<org.apache.zeppelin.helium.ClassLoaderApplication: org.apache.zeppelin.helium.Application app> >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r8 = r0.<org.apache.zeppelin.helium.ClassLoaderApplication: org.apache.zeppelin.helium.Application app> >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r8.<org.apache.zeppelin.helium.Application: void run(org.apache.zeppelin.resource.ResourceSet)>(r1)
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r8.<org.apache.zeppelin.helium.Application: void run(org.apache.zeppelin.resource.ResourceSet)>(r1) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r10 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r8.<org.apache.zeppelin.helium.Application: void run(org.apache.zeppelin.resource.ResourceSet)>(r1) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r11 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r8.<org.apache.zeppelin.helium.Application: void run(org.apache.zeppelin.resource.ResourceSet)>(r1) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r8.<org.apache.zeppelin.helium.Application: void run(org.apache.zeppelin.resource.ResourceSet)>(r1) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r9 = staticinvoke <java.lang.Thread: java.lang.Thread currentThread()>()
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r9 = staticinvoke <java.lang.Thread: java.lang.Thread currentThread()>() >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r9.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>(r3)
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r9.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>(r3) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : goto [?= return]
org.apache.zeppelin.helium.ClassLoaderApplication.run : goto [?= return] >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : return
org.apache.zeppelin.helium.ClassLoaderApplication.run : r5 = $r14 >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r15 = staticinvoke <java.lang.Thread: java.lang.Thread currentThread()>()
org.apache.zeppelin.helium.ClassLoaderApplication.run : r5 = $r14 >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r15 = staticinvoke <java.lang.Thread: java.lang.Thread currentThread()>() >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r15.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>(r3)
org.apache.zeppelin.helium.ClassLoaderApplication.run : virtualinvoke $r15.<java.lang.Thread: void setContextClassLoader(java.lang.ClassLoader)>(r3) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : throw r5
org.apache.zeppelin.helium.ClassLoaderApplication.run : r12 = $r11 >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r13 = new org.apache.zeppelin.helium.ApplicationException
org.apache.zeppelin.helium.ClassLoaderApplication.run : r12 = $r11 >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r13 = new org.apache.zeppelin.helium.ApplicationException >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : specialinvoke $r13.<org.apache.zeppelin.helium.ApplicationException: void <init>(java.lang.Exception)>(r12)
org.apache.zeppelin.helium.ClassLoaderApplication.run : $r13 = new org.apache.zeppelin.helium.ApplicationException >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : specialinvoke $r13.<org.apache.zeppelin.helium.ApplicationException: void <init>(java.lang.Exception)>(r12) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : throw $r13
org.apache.zeppelin.helium.ClassLoaderApplication.run : specialinvoke $r13.<org.apache.zeppelin.helium.ApplicationException: void <init>(java.lang.Exception)>(r12) >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : throw $r13 >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : r4 = $r10 >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : throw r4
org.apache.zeppelin.helium.ClassLoaderApplication.run : r4 = $r10 >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.ClassLoaderApplication.run : throw r4 >>>>>>>> org.apache.zeppelin.helium.ClassLoaderApplication.run : $r14 := @caughtexception
